anaconda.spec.in

```diff
@@ -102,6 +102,10 @@ Requires: python3-systemd
 Requires: python3-productmd
 Requires: python3-dasbus >= %{dasbusver}
 Requires: flatpak-libs
+# dependencies for rpm-ostree payload module
+Requires: rpm-ostree
```

Comment:
I don't see where `rpm-ostree` and `skopeo` are used. Is it transparently called by another command?

Reply:
Yes, today the command `ostree container` is actually forwarded to rpm-ostree.  (This is confusing I know, but it's to avoid duplicating the Rust code for containers...)

Reply:
`ostree container image deploy` calls `skopeo` and I think also `rpm-ostree` is called by `ostree`. I took that dependency from Lorax. Might be worth checking why it's not `ostree`...